cli/utils.go

```diff
@@ -70,6 +73,57 @@ func parseMatchers(inputMatchers []string) ([]labels.Matcher, error) {
 	return matchers, nil
 }
 
+// GetRemoteAlertmanagerConfigStatus returns status responsecontaining configuration from remote Alertmanager
+func GetRemoteAlertmanagerConfigStatus(ctx context.Context, alertmanagerURL *url.URL) (*client.ServerStatus, error) {
```

Comment:
This should be lowercase, since it's not being used outside of this package. It can be exported later if the need arises.